bench/wtperf/wtperf.c

```diff
@@ -1499,12 +1499,6 @@ scan_worker(void *arg)
         }
 
     while (!wtperf->stop) {
-        /* Break the sleep up, so we notice interrupts faster. */
-        for (i = 0; i < opts->scan_interval; i++) {
-            sleep(1);
-            if (wtperf->stop)
-                break;
-        }
```

Comment:
Is there a particular reason why we're removing these lines?

Reply:
I wasn't too sure of the intention with this, but now looking again it does appear worth keeping. I can re-add this